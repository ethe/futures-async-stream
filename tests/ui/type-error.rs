// SPDX-License-Identifier: Apache-2.0 OR MIT

#![feature(coroutines)]

use futures_async_stream::stream;

#[stream(item = i32)]
async fn foo() {
    let a: i32 = "a"; //~ ERROR mismatched types
    yield 1;
}

fn main() {}
