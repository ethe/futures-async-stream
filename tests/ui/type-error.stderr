error[E0308]: mismatched types
 --> tests/ui/type-error.rs:9:18
  |
9 |     let a: i32 = "a"; //~ ERROR mismatched types
  |            ---   ^^^ expected `i32`, found `&str`
  |            |
  |            expected due to this
